cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,179 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
```

Comment:
This test case is not clear. It's better to give a meaningful name to the test case that describes what it is testing. For example, it could be `test_upload_image_to_movie_shows_image_in_movie_session_detail` or something similar.